### ISSUE TITLE
LibWeb/LibGfx: SVG infinite Bézier curve on bandcamp.com

### ISSUE THREAD

**Issue by ADKaster** [SerenityOS/serenity#21281]
Loading bandcamp.com on current master (5e15c29e22cb4dfab8889bba59af63a033a8457b) crashes with:

```
VERIFICATION FAILED: isfinite(error) at /Users/andrew/Source/serenity/Userland/Libraries/LibGfx/Painter.cpp:2265
0   liblagom-core.0.0.0.dylib           0x00000001030d9ad4 ak_verification_failed + 136
1   liblagom-gfx.0.0.0.dylib            0x0000000103293f8c void AK::Queue<Gfx::Painter::for_each_line_segment_on_cubic_bezier_curve(Gfx::Point<float>, Gfx::Point<float>, Gfx::Point<float>, Gfx::Point<float>, AK::Function<void (Gfx::Point<float>, Gfx::Point<float>)>&)::SegmentDescriptor, 1000>::enqueue<Gfx::Painter::for_each_line_segment_on_cubic_bezier_curve(Gfx::Point<float>, Gfx::Point<float>, Gfx::Point<float>, Gfx::Point<float>, AK::Function<void (Gfx::Point<float>, Gfx::Point<float>)>&)::SegmentDescriptor>(Gfx::Painter::for_each_line_segment_on_cubic_bezier_curve(Gfx::Point<float>, Gfx::Point<float>, Gfx::Point<float>, Gfx::Point<float>, AK::Function<void (Gfx::Point<float>, Gfx::Point<float>)>&)::SegmentDescriptor&&) + 0
2   liblagom-gfx.0.0.0.dylib            0x0000000103299190 Gfx::Path::segmentize_path() + 696
3   liblagom-web.0.0.0.dylib            0x00000001042a8b78 AK::IterationDecision Web::TreeNode<Web::Layout::Node>::for_each_in_inclusive_subtree<Web::Layout::SVGFormattingContext::run(Web::Layout::Box const&, Web::Layout::LayoutMode, Web::Layout::AvailableSpace const&)::$_1>(Web::Layout::SVGFormattingContext::run(Web::Layout::Box const&, Web::Layout::LayoutMode, Web::Layout::AvailableSpace const&)::$_1) const + 1428
4   liblagom-web.0.0.0.dylib            0x00000001042a84ec Web::Layout::SVGFormattingContext::run(Web::Layout::Box const&, Web::Layout::LayoutMode, Web::Layout::AvailableSpace const&) + 528
5   liblagom-web.0.0.0.dylib            0x0000000104283f48 Web::Layout::FormattingContext::layout_absolutely_positioned_element(Web::Layout::Box const&, Web::Layout::AvailableSpace const&) + 348
6   liblagom-web.0.0.0.dylib            0x000000010426abb4 Web::Layout::BlockFormattingContext::parent_context_did_dimension_child_root_box() + 404
7   liblagom-web.0.0.0.dylib            0x000000010426d3c4 Web::Layout::BlockFormattingContext::layout_block_level_box(Web::Layout::Box const&, Web::Layout::BlockContainer const&, Web::Layout::LayoutMode, Web::CSSPixels&, Web::Layout::AvailableSpace const&) + 3628
8   liblagom-web.0.0.0.dylib            0x000000010426a5f8 Web::Layout::BlockFormattingContext::layout_block_level_children(Web::Layout::BlockContainer const&, Web::Layout::LayoutMode, Web::Layout::AvailableSpace const&) + 164
9   liblagom-web.0.0.0.dylib            0x00000001040734a4 Web::DOM::Document::update_layout() + 736
10  liblagom-core.0.0.0.dylib           0x00000001030b8e18 AK::Function<void ()>::operator()() const + 76
11  liblagom-core.0.0.0.dylib           0x00000001030b8310 Core::EventReceiver::dispatch_event(Core::Event&, Core::EventReceiver*) + 112
12  liblagom-core.0.0.0.dylib           0x00000001030cfe64 Core::ThreadEventQueue::process() + 432
13  liblagom-core.0.0.0.dylib           0x00000001030b13ec Core::EventLoop::spin_until(AK::Function<bool ()>) + 124
14  liblagom-web.0.0.0.dylib            0x0000000104302330 Web::Platform::EventLoopPluginSerenity::spin_until(JS::SafeFunction<bool ()>) + 184
15  liblagom-web.0.0.0.dylib            0x0000000104154f8c Web::HTML::EventLoop::spin_until(JS::SafeFunction<bool ()>) + 144
16  liblagom-web.0.0.0.dylib            0x00000001041e9460 Web::HTML::HTMLParser::handle_text(Web::HTML::HTMLToken&) + 908
17  liblagom-web.0.0.0.dylib            0x00000001041df6fc Web::HTML::HTMLParser::run() + 480
18  liblagom-web.0.0.0.dylib            0x00000001041e1490 Web::HTML::HTMLParser::run(AK::URL const&) + 124
19  liblagom-web.0.0.0.dylib            0x0000000104084cc8 Web::parse_document(Web::DOM::Document&, AK::Detail::ByteBuffer<32ul> const&) + 124
20  liblagom-web.0.0.0.dylib            0x000000010408add8 JS::SafeFunction<void (AK::Detail::ByteBuffer<32ul>)>::CallableWrapper<Web::load_document(AK::Optional<Web::HTML::NavigationParams>)::$_0>::call(AK::Detail::ByteBuffer<32ul>) + 116
21  liblagom-web.0.0.0.dylib            0x0000000104102c30 JS::SafeFunction<void (AK::Detail::ByteBuffer<32ul>)>::operator()(AK::Detail::ByteBuffer<32ul>) const + 176
22  liblagom-web.0.0.0.dylib            0x0000000104102a28 JS::SafeFunction<void ()>::CallableWrapper<Web::Fetch::Infrastructure::Body::fully_read(JS::Realm&, JS::SafeFunction<void (AK::Detail::ByteBuffer<32ul>)>, JS::SafeFunction<void (JS::GCPtr<Web::WebIDL::DOMException>)>, AK::Variant<AK::Empty, JS::NonnullGCPtr<JS::Object>>) const::$_0::operator()(AK::Detail::ByteBuffer<32ul> const&)::'lambda'()>::call() + 60
23  liblagom-web.0.0.0.dylib            0x00000001041569d4 Web::HTML::Task::execute() + 84
24  liblagom-web.0.0.0.dylib            0x0000000104155100 Web::HTML::EventLoop::process() + 76
25  liblagom-web.0.0.0.dylib            0x000000010430320c AK::Function<void ()>::CallableWrapper<Web::Platform::TimerSerenity::TimerSerenity()::$_0>::call() + 88
26  liblagom-core.0.0.0.dylib           0x00000001030b8e18 AK::Function<void ()>::operator()() const + 76
27  liblagom-core.0.0.0.dylib           0x00000001030b8310 Core::EventReceiver::dispatch_event(Core::Event&, Core::EventReceiver*) + 112
28  liblagom-core.0.0.0.dylib           0x00000001030cfe64 Core::ThreadEventQueue::process() + 432
29  liblagom-core.0.0.0.dylib           0x00000001030b2514 Core::EventLoopImplementationUnix::exec() + 44
30  liblagom-core.0.0.0.dylib           0x00000001030b12f0 Core::EventLoop::exec() + 80
31  WebContent                          0x0000000102b31b6c serenity_main(Main::Arguments) + 3860
32  WebContent                          0x0000000102bafc34 main + 216
33  dyld                                0x0000000187f27f28 start + 2236
SIGPIPE
```

**Comment by MacDue** [comment]
Cannot reproduce. Possibly fixed by #21897.

**Comment by awesomekling** [comment]
This appears to be fixed indeed :smile:
